sdks/java/core/src/main/java/org/apache/beam/sdk/values/TimestampedValue.java

```diff
@@ -43,9 +44,16 @@
  * @param <V> the type of the value
  */
 public class TimestampedValue<V> {
+  /**
+   * Returns a new {@link TimestampedValue} with the default timestamp,
+   * represented by {@link BoundedWindow#TIMESTAMP_MIN_VALUE}.
+   */
+  public static <V> TimestampedValue<V> of(V value) {
+    return of(value, BoundedWindow.TIMESTAMP_MIN_VALUE);
+  }
 
   /**
-   * Returns a new {@code TimestampedValue} with the given value and timestamp.
+   * Returns a new {@link TimestampedValue} with the given value and timestamp.
```

Comment:
Probably not necessary to `{@link}` to the current class: http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide